app/index.js

```diff
@@ -1,7 +1,14 @@
 import 'whatwg-fetch';
+<<<<<<< HEAD
+// import './exercise/create-message-item';
+// import './exercise/add-message-item';
+// import './exercise/reverse-list';
+// import './exercise/sum-from-list';
+=======
 // Import `data.json`
 // Import `addGamesToList` from `twitch/add-games-to-list`
 
 
 // Look up a `ul` element with a class called `results`
 // Add games to the results list based on the data from `data.json`
+>>>>>>> eb1ab13731653051706223d3532d706ecdfac751
```

Comment:
Then remove this line which is showing the end of changes for things that you merged in from my branch.